packages/financial-templates-lib/test/clients/OptimisticOracleClient.js

```diff
@@ -104,34 +105,35 @@ describe("OptimisticOracleClient.js", function () {
     // Set a non-0 final fee for the collateral currency.
     await store.methods.setFinalFee(collateral.options.address, { rawValue: finalFee }).send({ from: accounts[0] });
 
-    optimisticOracle = await OptimisticOracle.new(liveness, finder.options.address, timer.options.address).send({
```

Comment:
I re-structured these tests to be a bit cleaner by correctly using the before each.